src/main/java/com/autotune/analyzer/services/ListExperiments.java

```diff
@@ -109,6 +112,7 @@ public boolean shouldSkipClass(Class<?> aClass) {
         }
         response.getWriter().println(gsonStr);
         response.getWriter().close();
+        listExp.stop(MetricsConfig.timerlistExp);
```

Comment:
We need try / catch / finally blocks in all places where we are calling these methods to ensure that there are no "hanging" timers that are still running